src/LanguageServer/ExternalAccess/Copilot/Handler/RequestContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.LanguageServer.ExternalAccess.Copilot;
+
+internal readonly struct RequestContext(LspRequestContext context)
+{
+    /// <inheritdoc cref="LspRequestContext.Workspace"/>
+    internal Workspace? Workspace => context.Workspace;
```

Comment:
I would prefer we not expose the workspace

Reply:
agreed, workspace should be avoided wherever possible.  it usually does not make sense to use it

Reply:
fixed